### ISSUE TITLE
Please, update posts/gradle-cpp to use `model.components`

### ISSUE THREAD

**Issue by jan-hudec** [carlosvin/carlosvin.github.io#1]
In commit https://github.com/carlosvin/cpp_gradle/commit/74620acbf82ebb6a60728d381bc67d037483678f you updated the [`build.gradle`](https://github.com/carlosvin/cpp_gradle/blob/master/build.gradle) to use

```groovy
model {
    components {
        hello(NativeLibrarySpec) …
        main(NativeExecutableSpec) …
    }
}
```

as also matches the Gradle documentation, but the text at https://carlosvin.github.io/en/posts/gradle-cpp.html still talks about `libraries` and `executables`.

Please, would you be so kind and bring them in sync?

**Comment by carlosvin** [comment]
Thanks for the feedback, you are absolutely correct, I'm updating it right now, I'll close this item when it is done.

**Comment by carlosvin** [comment]
fixed at commit #b357275
